helpers/slices/strings.go

```diff
@@ -0,0 +1,37 @@
+package slices
```

Comment:
I've seen some projects us `pkg` as the root for utility packages that are usable outside of the project.  See https://github.com/docker/docker/tree/master/pkg,  https://github.com/coreos/etcd/tree/master/pkg and https://github.com/kubernetes/kubernetes/tree/master/pkg.  They sometimes put a `README.md` in there as well.  I kind of like `pkg` as opposed to `helpers`.